addons/event/models/event_event.py

```diff
@@ -579,7 +579,7 @@ def action_set_done(self):
         if first_ended_stage:
             self.write({'stage_id': first_ended_stage.id})
 
-    def mail_attendees(self, template_id, force_send=False, filter_func=lambda self: self.state != 'cancel'):
+    def mail_attendees(self, template_id, force_send=False, filter_func=lambda self: self.state not in ('cancel', 'draft')):
```

Comment:
We could just update the call to it and left it untouched in stable (api change we can avoid).